web/src/main/java/nl/tehdreamteam/se42/web/soap/conversation/SoapConversationController.java

```diff
@@ -0,0 +1,32 @@
+package nl.tehdreamteam.se42.web.soap.conversation;
+
+import nl.tehdreamteam.se42.domain.Message;
+import nl.tehdreamteam.se42.web.controller.ConversationController;
+import nl.tehdreamteam.se42.web.soap.SoapWebServiceConstants;
+
+import javax.jws.WebParam;
+import javax.jws.WebService;
+import java.util.List;
+
+/**
+ * Webservice for controlling Soap request concerning {@code Conversations}.
+ *
+ * @author Oscar de Leeuw
+ */
+@WebService(targetNamespace = SoapWebServiceConstants.DEFAULT_NAMESPACE + "conversation")
+public class SoapConversationController implements ConversationController {
+    @Override
+    public void addMessage(@WebParam(name = "token") String token, @WebParam(name = "content") String content, @WebParam(name = "conversation_id") long conversationId) {
+
+    }
+
+    @Override
+    public void addUserToConversation(@WebParam(name = "token") String token, @WebParam(name = "user_id") long userId, @WebParam(name = "conversation_id") long conversationId) {
+
+    }
+
+    @Override
+    public List<Message> getMessages(@WebParam(name = "token") String token, @WebParam(name = "conversation_id") long conversationId) {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1168)